website/docs/r/cloud_awsGovcloud_link_account.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: "newrelic"
+page_title: "New Relic: newrelic_awsGovcloud_ink_account"
+sidebar_current: "docs-newrelic-resource-awsGovCloud-link-account"
+description: |-
+  Link an AwsGovCloud account to New Relic.
+---
+
+# Resource: newrelic_cloud_awsGovCloud_link_account
+
```

Comment:
Can you add an Important block here to let customers know we don't have access to a AWS GovCloud account and can't properly test this resource? Take a look at [entity_tags docs](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/entity_tags) for an example